src/plugins/autoColumnSize/autoColumnSize.js

```diff
@@ -622,6 +625,13 @@ export class AutoColumnSize extends BasePlugin {
     privatePool.get(this).cachedColumnHeaders = this.hot.getColHeader();
   }
 
+  onAfterFormulasValuesUpdate(changes) {
```

Comment:
Ohh, It seems that the linter didn't catch missing hook docs 🤔  Can you complete @Siemienik?

Reply:
done in: 424ef10